profile-widget/diveprofileitem.cpp

```diff
@@ -421,12 +421,15 @@ void DivePercentageItem::paint(QPainter *painter, const QStyleOptionGraphicsItem
 	QPolygonF poly = polygon();
 	for (int i = 1, modelDataCount = dataModel->rowCount(); i < modelDataCount; i++) {
 		if (i < poly.count()) {
-			double value = dataModel->index(i, vDataColumn).data().toDouble();
-			int cyl = dataModel->index(i, DivePlotDataModel::CYLINDERINDEX).data().toInt();
-			int inert = 1000 - get_o2(&displayed_dive.cylinder[cyl].gasmix);
-			mypen.setBrush(QBrush(ColorScale(value, inert)));
-			painter->setPen(mypen);
-			painter->drawLine(poly[i - 1], poly[i]);
+			double value = dataModel->index(i - 1, vDataColumn).data().toDouble();
+			int sec = dataModel->index(i - 1, hDataColumn).data().toInt();
+			if (value >= 1.0 && sec > 0) {
+				int cyl = dataModel->index(i, DivePlotDataModel::CYLINDERINDEX).data().toInt();
```

Comment:
so what happens in the heatmap if value < 1?
again, this close before the release, a better explanation either here in a comment or in the commit message would be useful. Of course, part of the problem is that "value" is maybe not the best variable name in the world for this...